HucaresWF/src/Hucares.Server.Client/MissingPlateClient.cs

```diff
@@ -0,0 +1,159 @@
+using System;
+using System.Collections.Generic;
+using System.Net.Http;
+using System.Text;
+using System.Threading.Tasks;
+using Hucares.Server.Client.Models;
+using Newtonsoft.Json;
+
+namespace Hucares.Server.Client
+{
+    public class MissingPlateClient : IMissingPlateClient
+    {
+        private Uri HostUri { get; set; } = new Uri("http://localhost:50510");
+        
+        private readonly HttpClientHelper httpHelper;
+        
+        public MissingPlateClient(HttpClientHelper httpHelper = null)
+        {
+            this.httpHelper = httpHelper ?? new HttpClientHelper();
+        }
+        
+        public async Task<MissingLicensePlate> InsertPlateRecord(string plateNumber, DateTime searchStartDatetime)
+        {
+            var uri = $"api/mlp/insert/{plateNumber}";
```

Comment:
This is a mistake in both the backend. An insert should not have plateNumber in uri, because it is already in the body